src/components/Delegation/index.js

```diff
@@ -0,0 +1,55 @@
+//  @flow
+import React, { useState, useEffect } from 'react'
+import type { AccountLike, Account } from '@ledgerhq/live-common/lib/types'
+import { Trans } from 'react-i18next'
+import { loadAccountDelegation } from '@ledgerhq/live-common/lib/families/tezos/bakers'
+import { getMainAccount } from '@ledgerhq/live-common/lib/account'
+
+import Text from 'components/base/Text'
+import Box from 'components/base/Box'
+import Card from 'components/base/Box/Card'
+
+import Header from './Header'
+import Row from './Row'
+
+type Props = {
+  account: AccountLike,
+  parentAccount: ?Account,
+}
+
+const Delegation = ({ account, parentAccount }: Props) => {
+  const [delegation, setDelegation] = useState(null)
```

Comment:
TODO gre: provide `useDelegation` in live-common

```
const delegation = useDelegation(account)
```

so we can provide an initial value.